provider/gateway/client.go

```diff
@@ -117,6 +119,20 @@ func (c *client) ServiceStatus(ctx context.Context, host string, id mtypes.Lease
 	return &obj, nil
 }
 
+func (c *client) ServiceLogs(ctx context.Context, host string, id mtypes.LeaseID, service string) (*cluster.ServiceLogs, error) {
```

Comment:
I left this out because the backend implementation was a disaster.  We'll also have to use [gorilla/websocket](https://github.com/gorilla/websocket) in the server to stream the logs back.

but no worries, keep it here if you want to continue iterating on it.

Reply:
better to remove it then, as streamline implementation will be a bit different